_posts/2000-01-03-redundancy-payment.markdown

```diff
@@ -3,6 +3,7 @@
 layout: exemplar
 
 
+exemplar-service-url: "https://notice.redundancy-payments.org.uk/claims/start"
```

Comment:
@jabley Currently it has no start page since it is only a follow up service for people who have already claimed and they are informed by direct communication of the location to start.  The transaction is not one that anybody can start from GOV.UK, nor is it one they want people to find and start independently, such as via Google, which a start page would cause to happen.
That may change, but has been agreed with the GOV.UK team so far.